PeepPeep/PeepPeep/View/InitialSetupScreen/InitialSetupView.swift

```diff
@@ -4,16 +4,38 @@
 //
 //  Created by Ha Jong Myeong on 2023/07/12.
 //
+
 import SwiftUI
+import DeviceActivity
+import FamilyControls
 
 struct InitialSetupView: View {
+    @State var show = false
+    let center = AuthorizationCenter.shared
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        VStack {
+            if show {
+                ActivitySummaryView()
+            }
+            else {
+                STProgressView()
+            }
```

Comment:
```suggestion
            show ?  ActivitySummaryView() ? STProgressView()
```

삼항연산자를 사용하면 한 줄로 줄일 수 있어보여요 : )

Reply:
삼항 연산자를 바로 사용하니 뷰타입이 달라 오류가 생겼었네요 :)
`shouldDisplay ? AnyView(ActivitySummaryView()) : AnyView(STProgressView())`
AnyView 타입으로 래핑후 반영했습니다!! 감사합니다 😃